src/firefly/java/edu/caltech/ipac/astro/ibe/BaseIbeDataSource.java

```diff
@@ -57,7 +58,8 @@ public IbeQueryParam makeQueryParam(Map<String, String> queryInfo) {
 
         String userTargetWorldPt = queryInfo.get("UserTargetWorldPt");
         String upload = queryInfo.get("filename");
-        if (userTargetWorldPt != null || !isEmpty(upload)) {
+        String searchsid = queryInfo.get("sidsearch");
+        if (userTargetWorldPt != null || !isEmpty(upload) || searchsid != null ) {
```

Comment:
This addition is suspicious.  I assumed `sidsearch` mean scan id search. 
By adding `searchsid != null`, it will add `filename` and `size` to the parameters which is not applicable for scan id search.  Is this right?

Reply:
> This addition is suspicious. I assumed `sidsearch` mean scan id search. By adding `searchsid != null`, it will add `filename` and `size` to the parameters which is not applicable for scan id search. Is this right?

this added condition is for source id search only, it needs the ibe Intercects/mcen etc as url params, scanid don't need these params